api/src/utils/file-utils.ts

```diff
@@ -139,7 +140,7 @@ export interface IS3FileKey {
  * @return {*}  {string}
  */
 export function generateS3FileKey(options: IS3FileKey): string {
-  const keyParts: (string | number)[] = ['platform'];
+  const keyParts: (string | number)[] = [S3_KEY_PREFIX];
 
   if (options.submissionId) {
     keyParts.push(options.submissionId);
```

Comment:
```suggestion
    keyParts.push('submissions');
    keyParts.push(options.submissionId);
```

Sorry one extra change while we're editing this function.

Reply:
We have this pattern in sims, didn't get carried over for w/e reason.  Will help add some clarity to the path in S3.